### DISCUSSION TITLE
[Kavita] PDF converter to CBZ

### DISCUSSION THREAD

**Discussion by cristianconti** [Kareadita/Kavita#2873] [Ideas]
### Idea Description

Kavita should add the feature to convert PDF comic files to cbz.
A file type tag should be visible in comic listing pages (double tag for mixed file types in that folder). Converting this series to make all files cbz.
For a bulk conversion, can be a selection (maybe a collection) or a library conversion. 
A setting for a confirmation if the cbz final file size is bigger.
A setting for moving PDF files instead of deletion. 
Jpeg/webp setting will be a cherry on the top of the cake.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

8 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is outside the scope of Kavita's goal and fits with an external program.
